Projects/Addons/Sources/Models/AddonOffer.swift

```diff
@@ -58,6 +58,13 @@ public struct AddonOffer: Identifiable, Equatable, Hashable, Sendable {
         self.currentAddon = currentAddon
         self.quotes = quotes
     }
+
+    func getAddonPriceDiff(selectedQuote: AddonQuote?) -> MonetaryAmount? {
+        let currentAddonPrice = currentAddon?.price?.value
+        let newPrice = selectedQuote?.price?.value
+        let diffPrice = Int(newPrice ?? 0) - Int(currentAddonPrice ?? 0)
+        return MonetaryAmount(amount: diffPrice.asString, currency: selectedQuote?.price?.currency ?? "SEK")
+    }
 }
```

Comment:
Maybe rework this as
```
    func getTotalPrice(selectedQuote: AddonQuote?) -> MonetaryAmount? {
        guard let selectedQuote else { return nil}
        guard let currentAddon else { return selectedQuote.price }
        guard let currentAddonPrice = currentAddon.price,
              let newPrice = selectedQuote.price else { return nil }
        let diffPrice = newPrice.value - currentAddonPrice.value
        return MonetaryAmount(amount: diffPrice.asString, currency: newPrice.currency)
    }
```
Since this one can be used on both screens - when selecting addon and summary